jjb/teravm/templates/teravm-templates.yaml

```diff
@@ -0,0 +1,67 @@
+- job-template:
+    name: "{project-name}-LF-teravm"
+    node: "ci-teravm"
+    build-args: ""
+
```

Comment:
We need to add now in the job template the publisher

```
- job-template:
    name: '{name}-unit-tests'
    builders:
    - shell: unittest
    publishers:
    - email:
        recipients: '{mail-to}'
```

Ref. https://docs.openstack.org/infra/jenkins-job-builder/publishers.html#publishers.slack